src/understanding-workflow/concepts.md

```diff
@@ -23,14 +23,15 @@ the activity on your cluster, including: desired state convergence, stable
 service addresses, health monitoring, service discovery, and DNS resolution.
 
 Workflow builds upon Kubernetes abstractions like Services, Replication
-Controllers and Pods to provide a developer-friendly UX, source to image, log
-aggregation, etc.
+Controllers, and Pods to provide a developer-friendly experience. Building containers
+directly from application source code, aggregating logs, and managing deployment
+configurations and app releases are just some of the features Workflow adds.
 
-Workflow is shipped as a Kubernetes-native application, installable via
-[Helm][helm], so operators familiar with Kubernetes will feel right at home
+Deis Workflow is a set of Kubernetes-native components, installable via
+[Helm][helm]. Ops who are familiar with Kubernetes will feel right at home
```

Comment:
I think `Ops` needs to be more specific. Maybe `Ops professionals` or `Systems engineers`?